interface/scan_points.h

```diff
@@ -408,6 +408,50 @@ namespace scans {
     case 355127: return 45;
     case 355128: return 45;
     case 355129: return 45;
+	  // 2022
+      // Scan 3 - Sep 2022 mini scan
+      // http://cmsonline.cern.ch/cms-elog/1157556
+    case 359576: return 46;
+    case 359577: return 46;
+    case 359578: return 46;
+    case 359579: return 46;
+    case 359580: return 46;
+    case 359581: return 46;
+    case 359582: return 46;
+    case 359584: return 46;
```

Comment:
I am also noticing that the delay scan 46 is already on the `main` branch so whatever is being added here should not already be present on the `main` branch. Otherwise, this could potentially cause merge conflicts for PR#2.